README.md

```diff
@@ -56,16 +56,8 @@ access to any of the memory that RMM is holding.
 
 ## Qualification and Profiling tools
 
-The Qualification tool is used to look at a set of applications to determine if the RAPIDS Accelerator for Apache Spark
-might be a good fit for those applications.
-
-The Profiling tool generates information which can be used for debugging and profiling applications.
-Information such as Spark version, executor information, properties and so on. This runs on either CPU or
-GPU generated event logs.
-
-Please refer to [Qualification tool documentation](docs/spark-qualification-tool.md)
-and [Profiling tool documentation](docs/spark-profiling-tool.md)
-for more details on how to use the tools.
```

Comment:
We want to leave this comment in the core README as we are leaving documentation in the spark-rapids repo.

Reply:
Added it back.